src/docs/Components/Form/QuantitySelector/accessibility.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: 'Accessibility'
+order: 3
+---
+
+
+## Sizes
+
+Please know that the `small` version is really supposed to be used on touch devices to avoid misclicking. We recommend some blank space before and after the quantity selector to have the best user experience possible.
+
```

Comment:
> Please know that the `small` version is really supposed to be used on touch devices to avoid misclicking.

I don't understand this sentence. I have the impression that there is a "not" missing somewhere, isn't there?

Reply:
indeed